packages/callout-block/src/CalloutBlock.tsx

```diff
@@ -33,19 +48,24 @@ export const CalloutBlock: FC<CalloutBlockProps> = ({ appBridge }) => {
         }
     };
 
-    const textDivClassNames = joinClassNames([
-        'tw-flex tw-items-center',
-        isDark(getBackgroundColor(blockSettings.type)) ? 'tw-text-white' : 'tw-text-black',
-        blockSettings.width === Width.FullWidth && alignmentMap[blockSettings.alignment],
-        !blockSettings.hasCustomPadding && paddingMap[blockSettings.paddingChoice],
-    ]);
-
     const customPaddingStyle = {
         padding: blockSettings.hasCustomPadding
             ? `${blockSettings.paddingTop} ${blockSettings.paddingRight} ${blockSettings.paddingBottom} ${blockSettings.paddingLeft}`
             : '',
     };
 
+    const color = getBackgroundColor(blockSettings.type);
+    const backgroundColor = blockSettings.appearance === Appearance.Strong ? setAlpha(color, 1) : setAlpha(color, 0.1);
+
+    const defaultTextColor = isDark(color) ? 'white' : 'black';
+    const textColor = blockSettings.appearance === Appearance.Light ? setAlpha(color, 1) : defaultTextColor;
```

Comment:
Same here, like this the alpha value of the user (global design tokens) is ignored.